app/views/users/index.html.erb

```diff
@@ -1,2 +1,16 @@
-<h1> Here is list of users and their corresponding number of posts
-</h1>
+<div class="users-index-wrapper">
+<h1 class="header-text">Here is list of users and their corresponding number of posts</h1>
+
+<ul class="users-container">
+  <% @users.each do |user| %>
+    <li class="author-info user-link-container">
+      <img class="post-box" src ="/box-post.png" alt="Post logo">
+      <div class="user-link-wrapper">
+        <p class="username-link"> <%= link_to user.name, user_path(user) %> </p>
+        <p class="posts-counter"> Number of posts: <%= user.posts_counter %></p>
+      </div>
+    </li>
+  <% end %>
```

Comment:
- [ ] Good job! But you need to use some flex to layout the elements to match the given wireframe.  Right now, your index page looks like this: 
![image](https://github.com/obibaadoma/Blog_App/assets/91486791/211e9c65-cb2e-4507-87e0-63da69c4a77b)

It should look like the given wireframe. 
![image](https://github.com/obibaadoma/Blog_App/assets/91486791/9fb9e8b9-689d-46fb-a6b0-a83c73f47c76)

Reply:
added css